models/invite_institution.py

```diff
@@ -59,11 +61,17 @@ def send_email(self, host, body=None):
     def send_notification(self, user):
         """Method of send notification of invite institution."""
         entity_type = 'INSTITUTION'
-        super(InviteInstitution, self).send_notification(user, entity_type)
+
+        user_found = User.query(User.email == self.invitee).fetch(1)
```

Comment:
verificar se o usuário é ativo tambem, pois se for um user inativo a notificação é enviada em vão

Reply:
Verificado